### PR TITLE
Sprint 5 deploy to www-test

### PR DESCRIPTION
**Pull request by z3cka**
Deploying panelizer update as first part of sprint 5 QA to `www-test`. this solves DRUP-1235.

### CODE REVIEW

File: www/sites/all/modules/features/uclalib_staff_directory/uclalib_staff_directory.module [hunk 1]
```diff
@@ -18,7 +18,6 @@ function uclalib_staff_directory_page_alter(&$page) {
     // detect if an alias needs to be created
     if ($path == $alias) {
       $location_alias = drupal_get_path_alias('node/'. arg(1));
-      dpm($location_alias, "\$location_alias");
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_staff_directory/uclalib_staff_directory.module]: @akohler I removed this! FYI.

**Comment by z3cka** [comment]
This is going to `www-test`. @akohler let me know if you see anything out of whack.

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

File: www/sites/all/modules/features/uclalib_staff_directory/uclalib_staff_directory.module [hunk 2]
```diff
@@ -182,7 +181,7 @@ function uclalib_staff_directory_form_views_exposed_form_alter(&$form, &$form_st
 /**
 * Create Staff Directory Gallery default block.
 */
-function uclalib_staff_directory_update_7000(&$sandbox) {
+function uclalib_staff_directory_install() {
```
Comment (jameswilson) [inline www/sites/all/modules/features/uclalib_staff_directory/uclalib_staff_directory.module]: Typically we would do this slightly differently,  leaving uclalib_staff_directory_update_7000() as is, and adding a hook_install() function that then calls the uclalib_staff_directory_update_7000() function.

**Pull request merged**